Hymns/Common/BottomSheetView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+private enum Constants {
+    static let radius: CGFloat = 16
+    static let indicatorHeight: CGFloat = 6
+    static let indicatorWidth: CGFloat = 60
+    static let snapRatio: CGFloat = 1
+    static let minHeightRatio: CGFloat = 1
+}
+
+struct BottomSheetView<Content: View>: View {
```

Comment:
If you follow a tutorial or something to write, could you add a comment referencing it? That makes it easier to review too cuz then there are explanations for why things are done the way they are

Reply:
Yeah, good point. Added in 155b533